x-pack/plugins/security_solution/common/api/entity_analytics/common/common.schema.yaml

```diff
@@ -123,13 +94,30 @@ components:
           description: The weighted risk score of the risk input document.
         timestamp:
           type: string
-          example: '2017-07-21T17:32:28Z'
+          example: "2017-07-21T17:32:28Z"
           description: The @timestamp of the risk input document.
+        contribution_score:
+          type: number
+          format: double
+
+    RiskScoreCategories:
+      type: string
+      enum:
+        - category_1
+
+    EntityRiskLevels:
+      type: string
+      enum:
+        - "Unknown"
+        - "Low"
+        - "Moderate"
+        - "High"
+        - "Critical"
 
-    RiskScore:
+    EntityRiskScore:
```

Comment:
nit: I'm not well familiar with a business domain here but renaming to `EntityRiskScore` maybe not the best since it doesn't convey what's different with rule's `RiskScore` schema which is a number. Looking at the properties here it looks for me like `RiskScoreCalculationResult` os something like that. Naming schemas according to the business domain is always a good practice.

Reply:
"RiskScore" is a number in the rules domain, but in the Entity analytics domain, it is much more complex because we store the type of score, when it happened and every input that contributed to the score value. It is represented by an object with several properties, such as weight, inputs, timestamp, normalized score, and denormalized score. I named the type `EntityRiskScoreRecord` so, it is less confusing.